views/hooks/useChartsList.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { useOutletContext } from 'react-router-dom'
+import api from '../api'
+
+const NULL_CHART = {}
+
+export default function useChartsList() {
+  const { user, setNotification, users } = useOutletContext()
+  const [chartToShare, setChartToShare] = React.useState(NULL_CHART)
```

Comment:
Can we import `useState` at the top so we aren't calling `React.useState` everywhere?